batch/job_launcher.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+import boto3
+import click
+import glob
+import os
+import re
+import tarfile
+import time
+import yaml
+
+@click.command()
+@click.option("-p", "--job-prefix", type=str, required=True,
```

Comment:
nit: intelligent defaults to reduce the number of required params would be nice. e.g., `job-prefix` is `name` in config file plus a uniqifer, `num-jobs` is calculated based on # of scenarios/# of sims with some cap, etc.